webgui-new/README.md

```diff
@@ -0,0 +1,76 @@
+# New Next.js (React) based webgui for CodeCompass
```

Comment:
We could name this *New Web GUI development guide* and focus only on the parts on how to set up a development environment for the new web GUI. If someone would not like to develop the web GUI, it should be enough to read just the regular development guide.